### PR TITLE
Added new configuration parameter 'Verify SSL'

### PR DESCRIPTION
**Pull request by cs-sameer-pathan**
**PMDB ID**: 30773
#### Changes:
- Added new configuration parameter 'Verify SSL'
- Updated copyright.
- Updated release notes.

#### UTC:
- [x] Tested and verified connector health.
- [x] Verified connector version.

### CODE REVIEW

**Review by cs-prerna-joshi — APPROVED** [review]

File: openai/operations.py [hunk 1]
```diff
@@ -58,30 +57,33 @@ def _build_messages(params):
...
             "api_type": "azure",
             "api_version": config.get("api_version")
         })
-    https_proxy = os.environ.get('HTTPS_PROXY')
-    no_proxy = os.environ.get('NO_PROXY', 'localhost')
-    if https_proxy and (not (openai.base_url and openai.base_url in no_proxy) or ('api.openai.com' in no_proxy)):
-        openai_client = openai.OpenAI(http_client=httpx.Client(proxy=https_proxy, verify=False), **openai_args)
+        base_url = config.get("api_base")
+    verify_ssl = config.get('verify_ssl')
+    if https_proxy and base_url not in no_proxy:
```
Comment (cs-pooja-singh) [inline openai/operations.py:78]: what if azure endpoint as base_url?

File: openai/operations.py [hunk 2]
```diff
@@ -142,7 +144,7 @@ def check(config):
         list_models(config, {})
         return True
     except Exception as err:
-        logger.error('{0}'.format(err))
+        logger.exception('{0}'.format(err))
```
Comment (cs-pooja-singh) [inline openai/operations.py:147]: not required. in the execute function in operations.py, we are already logging the same trace

File: openai/release_notes.md [hunk 3]
```diff
@@ -1,9 +1,4 @@
...
-  - Converse With OpenAI
-  - List Available Models
-  - Get Tokens Usage
-  - Get Token Count
-- Renamed the `Create a chat completion` action and playbook to `Ask a Question`.
-- Added `Timeout` parameter in `Ask a Question` action.
-- Enhanced connector to be able to connect to Azure OpenAI Service. A new configuration parameter named 'Use Microsoft Azure OpenAI Endpoint' is added for users who want to use OpenAI services via a Microsoft Azure Endpoint. 
\ No newline at end of file
+- Added a new configuration parameter `Verify SSL`.
+- Initialized httpx client with proxy if proxy is configured in the environment.
```
Comment (cs-pooja-singh) [inline openai/release_notes.md:3]: Added Network Proxy support. Call to openai endpoint will be routed via the network proxy if proxy is configured for the FortiSOAR instance

File: openai/operations.py (hunk 1 above)
Comment (cs-sameer-pathan) [reply openai/operations.py:78]: If azure endpoint as base_url updated base_url

File: openai/operations.py (hunk 2 above)
Comment (cs-sameer-pathan) [reply openai/operations.py:147]: In connectors.py we logged stacktrace only in execute, provided for check_healt

**Pull request merged**